opengrok-indexer/src/test/java/org/opengrok/indexer/configuration/MetricsConfigureTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+ */
+
+package org.opengrok.indexer.configuration;
+
+import io.micrometer.graphite.GraphiteProtocol;
+import io.micrometer.statsd.StatsdFlavor;
+import org.junit.Test;
+import org.opengrok.indexer.Metrics;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Doesn't `*` get disapproval?

Reply:
yes, I neglected that. We should really have that checked as part of the build (filed #3298).